package.json

```diff
@@ -43,12 +43,13 @@
     "react-tap-event-plugin": "^1.0.0"
   },
   "dependencies": {
+    "babel-runtime": "^6.11.6",
```

Comment:
Yes, it's needed: https://github.com/oliviertassinari/react-swipeable-views/issues/149.